browser/src/slideshow/engine/SlideShowNavigator.ts

```diff
@@ -194,8 +194,11 @@ class SlideShowNavigator {
 				', bSkipTransition: ' +
 				bSkipTransition,
 		);
-		if (nNewSlide === undefined || nNewSlide < 0) return;
-		if (nNewSlide >= this.theMetaPres.numberOfSlides) {
+		if (
```

Comment:
I don't think we should quit if user try to go back from first slide.
Anyhow I've not yet the time to understand what's the root cause of the problem.

Reply:
`this.currentSlide === nNewSlide